stimpacks/aws/login.go

```diff
@@ -0,0 +1,161 @@
+package aws
+
+import (
+	"github.com/hashicorp/vault/api"
+	"github.com/readytalk/stim/pkg/log"
+	"github.com/skratchdot/open-golang/open"
+
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+var stimURL = "https://github.com/ReadyTalk/stim"
```

Comment:
Dont really like having this hardcoded here.

Reply:
Sure will have a TODO, like talked about above